zbus/src/connection/socket/command.rs

```diff
@@ -0,0 +1,126 @@
+use std::io;
+use std::os::fd::BorrowedFd;
+
+#[cfg(feature = "tokio")]
+use tokio::io::{AsyncReadExt, ReadBuf};
+
+use crate::process::{Child, ChildStdin, ChildStdout};
+
+use super::{ReadHalf, RecvmsgResult, Socket, Split, WriteHalf};
+
+/// A Command stream socket.
+///
+/// This socket may be used for communicating with a spawned child process via
```

Comment:
Let's not give an illusion of choice here: `may be used for communicating` -> `communicates`.  :)